unity-client/Assets/Rendering/Culling/CullingController.cs

```diff
@@ -150,31 +204,11 @@ internal IEnumerator ProcessProfile(CullingControllerProfile profile)
         /// </summary>
         IEnumerator UpdateCoroutine()
         {
-            RaiseDataReport();
-            profiles = new List<CullingControllerProfile> {settings.rendererProfile, settings.skinnedRendererProfile};
-            CommonScriptableObjects.rendererState.OnChange += (current, previous) => SetDirty();
-
             while (true)
             {
-                if (!CommonScriptableObjects.rendererState.Get())
-                {
-                    timeBudgetCount = 0;
-                    yield return null;
-                    continue;
-                }
-
-                bool shouldCheck = sceneObjects.dirty;
+                bool shouldCheck = objectsTracker.IsDirty() | playerPositionDirty;
```

Comment:
Just out of curiosity, is there any benefit of using bitwise vs logical operations with booleans?

Reply:
In this case not really, this was just a style choice. It may be a bad idea because I'm not sure if the expression is short-circuited (i.e. stopped being evaluated if the left side is true). I guess It's better to change it just in case.